appimage/build_appimage.sh

```diff
@@ -123,6 +124,7 @@ clone_monkey_repo() {
   fi
 
   log_message "Cloning files from git"
+  git config --global core.autocrlf false
   git clone --single-branch --recurse-submodules -b "$branch" "$MONKEY_ORIGIN_URL" "$repo_dir" 2>&1 || handle_error
```

Comment:
This change will actually modify the user's `~/.gitconfig` and then affect any clones for all subsequent repositories.

What about this instead:
```suggestion
  git clone -c core.autocrlf=false --single-branch --recurse-submodules -b "$branch" "$MONKEY_ORIGIN_URL" "$repo_dir" 2>&1 || handle_error
```

@VakarisZ I haven't tested that since I don't have WSL set up. Can you test and make the change if it works?